x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/lang_ident/lang_ident_inference.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+
+import { trainedModelsApiProvider } from '../../../../../services/ml_api_service/trained_models';
+
+const DEFAULT_INPUT_FIELD = 'text_field';
+
+export type FormattedLangIdentResp = Array<{
+  className: string;
+  classProbability: number;
+  classScore: number;
+}>;
+
+export class LangIdentInference {
```

Comment:
I'd prefer having a general Kibana API endpoint for testing models input that receives a model type as a request param and calls an appropriate endpoint

Reply:
I kept the endpoints separate on purpose as it is better for debugging. The endpoints being called are just basic wrappers for es ones.
The pytorch models just use the basic [infer](https://www.elastic.co/guide/en/elasticsearch/reference/current/infer-trained-model-deployment.html) endpoint and the lang ident uses a basic pipeline simulate.

I plan to create basic JSON input/output components that could be used for switching the "JSON mode" for each testable model. They should be as transparent as possible IMO and that means calling the es endpoints (or basic wrappers of) directly.
A user may wish to copy the ingest pipeline processor created in this component and add it to a real pipeline for ingest.

By having them behind a single endpoint, we'd have to manipulate the input and output of unrelated model types so they are the same. Or have the endpoint accept completely different input/outputs based on type, in which case why have a single endpoint.